roles/openshift/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+# Common variables
+openshift_env: 'test'
+openshift_roles: [ 'master', 'node' ]
+openshift_bind_ip: "{{ ansible_default_ipv4.address }}"
+openshift_public_ip: "{{ ansible_default_ipv4.address }}"
```

Comment:
Not necessarily development instances, more like small scale test or pre-POC instances.